example/ai-requestor/.env

```diff
@@ -115,8 +115,10 @@ YA_NET_BIND_URL=udp://0.0.0.0:11503
 # Address of relay server.
 # YA_NET_RELAY_HOST=127.0.0.1:7464
 # YA_NET_RELAY_HOST="yacn2a.dev.golem.network:7477" # second relay server for dev testing
```

Comment:
this comment can be removed, or changed tp be `yacn2` as it is the same as the next line now

Reply:
Removed.